content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -423,6 +424,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
    use EndpointSlices as the primary data source instead of Endpoints, enabling
    scalability and performance improvements. See [Enabling Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpointslices/).
 - `GCERegionalPersistentDisk`: Enable the regional PD feature on GCE.
+- `SetHostnameAsFQDN`: Enable the ability of setting Fully Qualified Domain Name(FQDN) as hostname of pod. See [Pod's `setHostnameAsFQDN` field](/docs/concepts/services-networking/dns-pod-service/#pod-sethostnameasfqdn-field).
```

Comment:
nit: Move this item so that the names are alphabetical?

Reply:
yes, I moved it